### ISSUE TITLE
Several terms needed for ACEnano KI dev are missing from ENM ontology

### ISSUE THREAD

**Issue by UanaF** [enanomapper/ontologies#170]
For ACEnano Knowledge infrastructure we need the following terms which are not present in ENM ontology but they are present in the ontologies from which ENM ontology is composed. Could we add/link them? I am writing here also the link of the terms from the other ontologies, it might be useful.

Raw data		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C142663
Compound		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C43366
Medium		http://purl.bioontology.org/ontology/npo#NPO_1853
Pore size		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C112332
Speed		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C41146
Temperature		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C25206
Suspension		http://purl.bioontology.org/ontology/npo#NPO_532
Extraction		http://purl.obolibrary.org/obo/CHMO_0001577
Filtration		http://purl.obolibrary.org/obo/NCIT_C16583
Freeze-drying		http://purl.obolibrary.org/obo/NCIT_C28150
Freezing		http://purl.obolibrary.org/obo/NCIT_C48160
Storage temperature		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C101707
Hydrophobicity		http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C63813
Electrophoretic mobility		http://purl.bioontology.org/ontology/npo#NPO_1315
X-Ray photon spectroscopy		http://purl.obolibrary.org/obo/CHMO_0000404
pH		http://purl.obolibrary.org/obo/UO_0000196
DLS intrument		http://purl.bioontology.org/ontology/npo#NPO_1766
Cells		http://purl.obolibrary.org/obo/NCIT_C12508
Powder		http://purl.obolibrary.org/obo/NCIT_C45302
Laser induced breakdown detection		http://purl.obolibrary.org/obo/CHMO_0000258
Small-Angle X-ray Scattering		http://purl.obolibrary.org/obo/CHMO_0000204

I have more terms which need annotation but first maybe it is better to see how we can do with these ones. Would be okay if I start to add them based on the adding terms enm tutorial?https://github.com/enanomapper/tutorials/tree/master/Added%20ontology%20terms. Please let me know/keep me updated on the process.

**Comment by egonw** [comment]
Yes, most certainly okay and indeed very much appreciated to follow the tutorial and add some terms! I recommend making one pull request for each term, and start with one, so that I can give feedback.

**Comment by egonw** [comment]
@UanaF, let me know if I can assist.

**Comment by UanaF** [comment]
Hi Egon, 

Thank you for offering the help and sorry for my late reply! The tutorial is a v useful resource and I think I can manage; I will start these days to add the terms and notify you for a feedback. I had other urgent tasks lately but will soon finish them and start with ENM ontology terms. 

/Oana

Trimis de pe iPhone‑ul meu

Pe 14 iul. 2019, la 12:11, Egon Willighagen <notifications@github.com> a scris:

**Comment by UanaF** [comment]
Hi Egon,

I tried to add "raw data" (http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C142663) as a child of "data item" (http://purl.obolibrary.org/obo/IAO_0000027). I added in the "ncit.iris" the following: "+(http://purl.obolibrary.org/obo/IAO_0000027):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C142663 raw data". If I use bioportal to see if the term was added to ENM classes; "raw data" is not there. The same with "suspension" (http://purl.bioontology.org/ontology/npo#NPO_532) in npo.iris. Do the terms need approve from you to be seen in ENM or I am doing something wrong?

**Comment by egonw** [comment]
You added them to the development version of the ontology. You need to load that version to do such a check.

**Comment by egonw** [comment]
We were using a rather old version of NCIT: https://github.com/enanomapper/ontologies/commit/4c68e083750d47a731e3d43a9c0b08eb2baca0c7

**Comment by egonw** [comment]
`suspension` from NPO is actually correctly included.

**Comment by egonw** [comment]
@UanaF, for `Compound` use `http://purl.obolibrary.org/obo/CHEBI_23367` which is already in `ENM`.

**Comment by egonw** [comment]
For `pH` use `http://purl.bioontology.org/ontology/npo#NPO_1814` instead. See also https://github.com/enanomapper/ontologies/commit/75c1b35a3774b54513341692d1e592bce3bff501

**Comment by egonw** [comment]
Finally (for now), can you explain why you need `Temperature` for? There are various related terms, which may be the one to use.

**Comment by UanaF** [comment]
We need temperature for the Measurement protocol > Instrument Parameters;
when u set the instrument; which should be an alternative to it?

Oana C. Florean,

Research Scientist,

Edelweiss Connect GmbH

**Comment by egonw** [comment]
@UanaF, I would suggest [http://purl.bioontology.org/ontology/npo#NPO_1806](http://bioportal.bioontology.org/ontologies/ENM/?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2Fnpo%23NPO_1806&jump_to_nav=true).

**Comment by UanaF** [comment]
Ah, good, thanks!

Oana C. Florean,

Research Scientist,

Edelweiss Connect GmbH

**Comment by UanaF** [comment]
Can I have a link for this dev version?

Oana C. Florean,

Research Scientist,

Edelweiss Connect GmbH

**Comment by egonw** [comment]
It's in https://github.com/enanomapper/ontologies/blob/master/README.md

**Comment by UanaF** [comment]
Continuation of the new terms needed to be added

Chemical Methods Ontology (CHMO):

Ion-selective electrode	http://purl.obolibrary.org/obo/CHMO_0002393
Laser induced breakdown detection	http://purl.obolibrary.org/obo/CHMO_0000258
Time of flight secondary ion mass spectrometry	http://purl.obolibrary.org/obo/CHMO_0000565
Tip Enhanced Raman Scattering (nano-Raman)	http://purl.obolibrary.org/obo/CHMO_0000678
Transmission electron microscopy with electron energy loss spectroscopy	http://purl.obolibrary.org/obo/CHMO_0000373
Wide-Angle X-ray Scattering	http://purl.obolibrary.org/obo/CHMO_0000207


NanoParticle Ontology (NPO):

DLS intrument	http://purl.bioontology.org/ontology/npo#NPO_1766
Electrophoretic mobility	http://purl.bioontology.org/ontology/npo#NPO_1315
Medium	http://purl.bioontology.org/ontology/npo#NPO_1853
Phase	http://purl.bioontology.org/ontology/npo#NPO_1610


National Cancer Institute Thesaurus (NCIT)

Amplitude	http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C70831
Conductivity	http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C134263
Matrix-Assisted Laser Desorption/Ionization http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C48040
Parameters	http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C80474
Purity (resistivity)	http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C62352

**Comment by laurent2207** [comment]
Freeze-drying http://purl.obolibrary.org/obo/NCIT_C28150 --> Already in ENM? 

![image](https://user-images.githubusercontent.com/44968488/70630945-b17b2d80-1c2c-11ea-8a5e-5ad5517b2109.png)

https://bioportal.bioontology.org/ontologies/ENM/?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2Fnpo%23NPO_1957&jump_to_nav=true

**Comment by laurent2207** [comment]
Electrophoretic mobility http://purl.bioontology.org/ontology/npo#NPO_1315 --> see issue #184
